Azure/sonic-buildimage
# MACsec container and wpa_supplicant component

**- Why I did it**
Make SONiC support MACsec feature
The HLD about MACsec feature is at : [MACsec HLD](https://github.com/Azure/SONiC/blob/9edcf67d0e0095e9f831a94ddf4fbb4a267adf7b/doc/macsec/MACsec_hld.md)

**- How I did it**
Add MACsec container and [sonic-wpa-supplicant](https://github.com/Azure/sonic-wpa-supplicant/tree/wpa/hostap_2_9)

**- How to verify it**
This PR doesn't set MACsec container automatically start, You should manually start the container by `docker run docker-macsec`
wpa_supplicant binary can be found at MACsec container.
This PR depends on the PR, [WPA_SUPPLICANT](https://github.com/Azure/sonic-wpa-supplicant/pull/16), and The MACsec container will be set as automatically start by later PR.

**- A picture of a cute animal (not mandatory but encouraged)**
🐉
